lib/medtronic/medtronicSimulator.js

```diff
@@ -78,6 +83,18 @@ exports.make = function(config){
       }
       setCurrWizard(event);
     },
+    smbg: function(event) {
+      if(currSMBG != null && currSMBG.value === event.value) {
+        console.log('Duplicate SMBG value (',event.value,')',currSMBG.subType,currSMBG.time,'/',event.subType,event.time);
+        var duration = Date.parse(event.time) - Date.parse(currSMBG.time);
+        if ((duration < (15 * sundial.MIN_TO_MSEC)) && (event.subType === 'manual') && (currSMBG.subType === 'linked')) {
```

Comment:
Yes, at the moment it's exactly the same. I'll DRY it out at a future stage once the Animas mmol/L issue is resolved to prevent merge conflicts. In the meanwhile I'll just put in a TODO.